ui/src/containers/VolumePageContent.js

```diff
@@ -68,59 +100,78 @@ const VolumePageContent = (props) => {
     );
 
   // prepare the data for <PerformanceGraphCard>
-  const deviceName = volume?.status?.deviceName;
-  let instance;
-
-  if (!node.internalIP) {
-    // find the node name of this volume
-    const nodeName = volume?.spec?.nodeName;
-    const currentNode = nodes.find((node) => node.name === nodeName);
-    instance = `${currentNode?.internalIP}:${PORT_NODE_EXPORTER}`;
-  } else {
-    instance = `${node?.internalIP}:${PORT_NODE_EXPORTER}`;
-  }
-
-  const queryStartingTime = volumeStats?.queryStartingTime;
-  const volumeUsage = volumeStats?.volumeUsage?.find(
-    (vU) => vU.metric.persistentvolumeclaim === PVCName,
-  )?.values;
-  const volumeThroughputWrite = volumeStats?.volumeThroughputWrite?.find(
-    (vTW) =>
-      vTW.metric.instance === instance && vTW.metric.device === deviceName,
-  )?.values;
-  const volumeThroughputRead = volumeStats?.volumeThroughputRead?.find(
-    (vTR) =>
-      vTR.metric.instance === instance && vTR.metric.device === deviceName,
-  )?.values;
-  const volumeLatencyWrite = volumeStats?.volumeLatencyWrite?.find(
-    (vL) => vL.metric.instance === instance && vL.metric.device === deviceName,
-  )?.values;
-  const volumeLatencyRead = volumeStats?.volumeLatencyRead?.find(
-    (vL) => vL.metric.instance === instance && vL.metric.device === deviceName,
-  )?.values;
-  const volumeIOPSRead = volumeStats?.volumeIOPSRead?.find(
-    (vIOPSR) =>
-      vIOPSR.metric.instance === instance &&
-      vIOPSR.metric.device === deviceName,
-  )?.values;
-  const volumeIOPSWrite = volumeStats?.volumeIOPSWrite?.find(
-    (vIOPSW) =>
-      vIOPSW.metric.instance === instance &&
-      vIOPSW.metric.device === deviceName,
-  )?.values;
-  const volumeMetricGraphData = {
-    volumeUsage,
-    volumeThroughputWrite,
-    volumeThroughputRead,
-    volumeLatencyWrite,
-    volumeLatencyRead,
-    volumeIOPSRead,
-    volumeIOPSWrite,
-    queryStartingTime,
-  };
+  // const deviceName = volume?.status?.deviceName;
+  // let instance;
+
+  // if (!node.internalIP) {
+  //   // find the node name of this volume
+  //   const nodeName = volume?.spec?.nodeName;
+  //   const currentNode = nodes.find((node) => node.name === nodeName);
+  //   instance = `${currentNode?.internalIP}:${PORT_NODE_EXPORTER}`;
+  // } else {
+  //   instance = `${node?.internalIP}:${PORT_NODE_EXPORTER}`;
+  // }
+
+  // const queryStartingTime = volumeStats?.queryStartingTime;
+  // const volumeUsage = volumeStats?.volumeUsage?.find(
+  //   (vU) => vU.metric.persistentvolumeclaim === PVCName,
```

Comment:
The reason I commented this is because all this code will be re-enabled when I will add the metrics tab. I wanted to prevent this from being reviewed as "new code" since it was already there and it is a big chunk of code.

Should I delete it and re-add it later then ?

Reply:
IMO it makes sense to keep the code, even though we don't directly use them in this PR. 
It brings a lot of noise by commenting on it or removing it.

Reply:
I agree :) . The issue was that eslint was making the whole build fail for unused variables if I left the code in place without commenting it.

Reply:
Indeed. we'll have ESLint error for the unused vars.
I would suggest you to locally disable the Rules with Inline Comments.

```javascript
  // eslint-disable-next-line no-unused-vars
  const volumeMetricGraphData = {
    volumeUsage,
    volumeThroughputWrite,
    volumeThroughputRead,
    volumeLatencyWrite,
    volumeLatencyRead,
    volumeIOPSRead,
    volumeIOPSWrite,
    queryStartingTime,
  };
```